.travis.yml

```
@@ -24,8 +24,8 @@ env:
         - secure: "md4fgPt9RC/sCoN5//5PcNHLUd9gWQGewV5hFpWW88MRTjxTng1Zfs8r7SqlF2AkEEepFfyzq0BEe9c3FMAnFbec3KmqdlQen4V8xDbLrcTlvkPlTrYGbAScUvdhhqojB//hMHoTD4KvxAv9CiUwFBO4hCMmj2buWHUbV9Ksu5WCW9mF/gkt/hIuYAU6Mbwt8PiYyMgUpzMHO1vruofcWRaVnvKwmBqHB0ae86D4/drpwn4CWjlM12WUnphT2bssiyPkw24FZtCN6kPVta6bLZKBxu0bZpw2vbXuUG+Yh19Q4mp8wNYT3XSHJf8Hl5LfujF48+cLWu+6rlCkdcelyVylhWLFc3rGOONAv4G8jWW2yNSz/bLQfJnMpd81fQEu5eySmFxB7mdB0uyKpvIG1jMJQ73LlYKakKLAPdYhMFyQAHoX9gvCE3S4QR95DBMi5gM/pZubOCcMLdjPHB5JKpJHSjxbOzyVwgmsUIEgd5Bi2vZvvYQXn1plk4xpQ3PhXc+/gi33bzY89mKcfOn0HJ2pD1vLqDXRCBsMCakoLZ0JB/6bacaz4FngbsGWuQ+I1cz20lJGL/MSi9bW1G7Uoidt3GXXWDmXrWt70vIXlLIxr8XV0Mu/rPbauGgWE+ZSYEfvdM5sP+FNF7vQ5de+Fkvzg5Z3tTfR+O1W+d7+vM4="
         - TWINE_USERNAME=Leonardo.Uieda
         # The file with the listed requirements to be installed by conda
-        - CONDA_REQUIREMENTS=requirements.txt
-        - CONDA_REQUIREMENTS_DEV=requirements-dev.txt
+        - CONDA_REQUIREMENTS=conda-requirements.txt
+        - CONDA_REQUIREMENTS_DEV=conda-requirements-dev.txt
         - CONDA_INSTALL_EXTRA="codecov twine"
```

What about adding `gmt=6.0.0rc4` here, and removing it from `requirements.txt`? I reckon we should still rename it to `conda-requirements.txt` though.

```suggestion
         - CONDA_INSTALL_EXTRA="codecov twine gmt==6.0.0rc4"
```

---

Yes, I think it makes more sense to put `gmt` in `CONDA_INSTALL_EXTRA`. 

As for the renaming, after we remove `gmt` from requirements.txt, I don't see any reason to rename these files. Users can use `pip -r requirements.txt` to install all pygmt dependencies.

---

Ok, I've placed `gmt` in `CONDA_INSTALL_EXTRA` and renamed things back. Just waiting for tests to pass. Oh, and just ignore the 'Now' bot for now :see_no_evil: